tests/vite-app/util/generateExampleTokens.ts

```diff
@@ -0,0 +1,99 @@
+import { JWK, JWS } from "node-jose"; // https://github.com/cisco/node-jose
```

Comment:
Shall we use this file: tests/installation-and-basic-usage/test-util.ts

Then we have one file that does this everywhere, and we can remove the node-jose dependency

Reply:
converted from node-jose to test-util.js, but had to patch `jwk.n` to work with webcrypto (remove leading 0x00 and base64url encode)